mailpoet/assets/js/src/wizard/steps/pitch_mss_step/first_part.tsx

```diff
@@ -17,6 +19,13 @@ function openMailPoetShopAndGoToTheNextPart(event, history, step: string) {
 function MSSStepFirstPart(): JSX.Element {
   const history = useHistory();
   const { step } = useParams<{ step: string }>();
+  const state = useSelector('getKeyActivationState')();
+
+  useEffect(() => {
+    if (state.isKeyValid === true) {
+      history.push(`/steps/${step}/part/3`);
```

Comment:
I'm not familiar with the flow and desired behavior, but you may consider `push` vs `replace` in regard to the history stack (and going back in the browser). I'm not sure which one is more desired here, just sharing my thoughts & approving it.